### ISSUE TITLE
Upgrade CCT to have the same features as CPT

### ISSUE THREAD

**Issue by gdbay** [CrocoBlock/suggestions#6867]
We've been working with JetEngine for almost 2 years and since that we did not use the CCT even when it comes to a more complex projects and new sections on our portal. CCT is made to optimise server load on DB by creating a separate tables instead of post types. But unfortunately due to lack of features this option become a bit useless. Could you please make this features to improve CCT? 🙏

1. There is no way of making taxonomies or any alternative to group CCT and make pages dedicated to listings of selected group. 
2. Having a single page require adding a post type, that totally destroy the core concept and idea of making posts without post type. 
3. The possibility to add relations between CCT and another CCT is missing. 
4. The possibility to change author of the CCT is very important 
5. Adding JetReview system to CCT or any other comment system possible would make a big difference. 

Hope you'll have a chance to improve CCT.

**Comment by MikeAGP** [comment]
The point of CCT is to be fully custom. So, you can just add a field to your CCT for categories/taxonomy for grouping. Make it a checkbox array and populate from a glossary. Also, you can create relation between CCT now.

**Comment by TotalKommando** [comment]
I agree. There are several times when I try to use a feature thats only shown working in tutorials with CPT.
Right now Im facing one issue with trying to setup a good flexible comment system, I was going to use JetReviews but it only allows one "review" per person per post / page and the reply chain is limited.

I was going to use WPDiscuz but its not loading on my CCT page.
